pkg/chunked/cache_linux.go

```diff
@@ -80,7 +80,9 @@ var (
 func (c *layer) release() {
 	runtime.SetFinalizer(c, nil)
 	if c.mmapBuffer != nil {
-		unix.Munmap(c.mmapBuffer)
+		if err := unix.Munmap(c.mmapBuffer); err != nil {
+			logrus.Debugf("Error Munmap: layer %q: %v", c.id, err)
```

Comment:
“warn”, in the 2 cases in this file, please — otherwise we might be leaking address space without explanation.